tests/wdio/matchScreenshotAndTestStandards-spec.js

```diff
@@ -0,0 +1,38 @@
+/* global describe, before, browser, it, expect, Terra */
+
+const accessibility = require('../../lib/wdio/services/TerraCommands/accessiblity').default;
+const visualRegressions = require('../../lib/wdio/services/TerraCommands/visual-regression').default;
+
+describe('matchScreenshotAndTestStandards', () => {
+  describe('full implementation', () => {
+    before(() => {
+      browser.url('/compare.html');
+      browser.setViewportSize(Terra.viewports('tiny')[0]);
+    });
+
+    Terra.should.matchScreenshotAndTestStandards();
```

Comment:
I don't have a good suggestion right now, but I think it would be worth thinking of a shorter name for this....

Reply:
`Terra.should`....
- `passAudit()`
- `auditElement()`
- `meetCompliance()`
- `meetStandards()`

Reply:
Fixed.

Reply:
matchAccessibleScreenshot maybe?

Reply:
I like validateElement maybe?  Means that it's validating a given element in terms of screenshot and anything else we want like accessibility.

Reply:
I'm good with validateElement!